### PR TITLE
Update/css tachyons

### PR DESCRIPTION
**Pull request by ClaudivanFilho**
#### What is the purpose of this pull request?
- Update css to use tachyons classes

#### How should this be manually tested?
https://claudivan2--storecomponents.myvtex.com/

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Header/components/TopMenu.js [hunk 1]
```diff
@@ -75,24 +75,27 @@ class TopMenu extends Component {
...
+        'fixed shadow-5 top-0 z-999': fixed,
       }
     )
+    const contentClasses = 'w-100 w-90-m w-80-xl center flex justify-center pb4 pv2-m pv6-l ph3-s ph7-m ph6-xl'
     return (
       <ReactResizeDetector handleWidth>
         {
           width => {
-            const mobileMode = width < 769 || (global.__RUNTIME__.hints.mobile && (!width || width < 769))
+            const mobileMode = width < 640 || (global.__RUNTIME__.hints.mobile && (!width || width < 640))
```
Comment (luciannojunior) [inline react/components/Header/components/TopMenu.js]: Essa mudança de 640 pra 769 foi por conta de algum problema que tuas mudanças introduziram?

File: react/components/Header/components/TopMenu.js [hunk 2]
```diff
@@ -75,24 +75,27 @@ class TopMenu extends Component {
 
   render() {
     const { logoUrl, logoTitle, fixed } = this.props
-    const classes = classNames(
-      'vtex-top-menu w-100 bg-white flex justify-center pb4 pv2-m pv6-l z-999',
```
Comment (luciannojunior) [inline react/components/Header/components/TopMenu.js]: Pra onde foram esses paddings?

File: react/components/Header/components/TopMenu.js (hunk 1 above)
Comment (ClaudivanFilho) [reply react/components/Header/components/TopMenu.js]: @luciannojunior  Foi mais pra seguir o padrão do tachyons.
```css
@media screen and (min-width: 20em) // 320px small
@media screen and (min-width: 40em) // 640px  medium
@media screen and (min-width: 64em) // 1024px  large
@media screen and (min-width: 80em) // 1280px  extra large
```

File: react/components/Header/components/TopMenu.js (hunk 2 above)
Comment (ClaudivanFilho) [reply react/components/Header/components/TopMenu.js]: @luciannojunior para o `contentClasses`, na linha abaixo

Comment (ClaudivanFilho) [reply react/components/Header/components/TopMenu.js]: @luciannojunior para o `contentClasses`, na linha abaixo

**Comment by andreldsa** [comment]
[Header | CSS Refact](https://trello.com/c/oQvuEbHi/41-header-css-refact)

**Pull request merged**